keyboards/kprepublic/bm40hsrgb/keymaps/wolff_abnt2/keymap.c

```diff
@@ -1,133 +1,147 @@
-/* Copyright 2021 lmlmask
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
```

Comment:
All your code should contain a license header if im not mistaken (doesn't need to be the exact same one you've removed)